cmd/server/cmd/server.go

```diff
@@ -99,18 +103,45 @@ func serverCmd(printf, fatalf shared.FormatFn) *cobra.Command {
 	// TODO: implement an option to specify how traces are reported (hardcoded to report to stdout right now).
 	serverCmd.PersistentFlags().BoolVarP(&sa.enableTracing, "trace", "", false, "Whether to trace rpc executions")
 
-	serverCmd.PersistentFlags().StringVarP(&sa.serviceConfigFile, "serviceConfigFile", "", "serviceConfig.yml", "Combined Service Config")
-	_ = serverCmd.MarkPersistentFlagFilename("serverConfigFile", "yaml", "yml")
+	serverCmd.PersistentFlags().StringVarP(&sa.configStoreURL, "configStoreURL", "", "",
```

Comment:
Any reason why these can't be explicit parameters without a URL scheme?

-fsStore <file system path>
-redisStore <host:port>